### PR TITLE
Updates for 3.28.0

### PR DESCRIPTION
**Pull request by radTuti**
## Description

Prepare for v3.28.0 release

- update versions
- add release note
- fix release scripts (cp #8658 )

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Review by sridhartigera — APPROVED** [review]

**Review by lwr20 — APPROVED** [review]

**Review by caseydavenport — APPROVED** [review]

File: release-notes/v3.28.0-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+The Calico CNI plugin can now optionally delay pod startup until after the local dataplane has been programmed. This may be desired in high churn scenarios to ensure network connectivity is avaialble from the moment the application software starts.
+
+#### Support for multiple IP pools in Tigera Operator installs
+
+The tigera operator now suppports configuring multiple IP pools at install time using the Installation API. IP pools can be modified after install via the Installation API as well.
+
+#### Bug fixes
+
+**Breaking change:** On upgrade, the UID of projectcalico.org/v3 resources will change. It is recommended that you restart any controllers that may care about this after upgrading Calico, including the kube-controller-manager. [calico #8586](https://github.com/projectcalico/calico/pull/8586) (@caseydavenport)
```
Comment (matthewdupre) [inline release-notes/v3.28.0-release-notes.md:17]: Can this one go in an "Important Upgrade Notes" section at the top of the whole release notes page as #2 (see other comment for #1) please

File: release-notes/v3.28.0-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+ - Fix confd issues when running on Windows operator installations (using HPC). [calico #8421](https://github.com/projectcalico/calico/pull/8421) (@coutinhop)
+ - Fixed AutoCreateServiceAccountTokenSecret param handling in install-calico-windows.ps1 [calico #8365](https://github.com/projectcalico/calico/pull/8365) (@coutinhop)
+ - Use KUBECONFIG env variable to build cluster config [calico #8549](https://github.com/projectcalico/calico/pull/8549) (@skmatti)
+
+#### Other changes
+
+**Important:** This release re-enables VXLAN checksum offload for kernels > 5.7 (this takes effect when nodes are restarted).  The offload was disabled in the past due to kernel bugs but has substantial performance impact.  We've been unable to find an environment where this needs to be disabled, so are re-enabling the offload with the intent of proving it works or finding a setup where it doesn't. If you encounter problems after upgrading and restarting nodes, please let us know and set "ChecksumOffloadBroken=true" in the FelixConfiguration's featureDetectOverride field to restore the previous behavior. [calico #8774](https://github.com/projectcalico/calico/pull/8774) (@tomastigera)
```
Comment (matthewdupre) [inline release-notes/v3.28.0-release-notes.md:44]: And this one at the top of the "Important Upgrade Notes" section

**Review by matthewdupre — COMMENTED** [review]
Couple of RN ordering tweaks

**Pull request merged**